### PR TITLE
Ft heatmap

### PR DESCRIPTION
**Pull request by Eder-PG**
Adding the full features of the Design teardown project

HTML validator and stickler show no errors or conflicts

Current live version:
https://raw.githack.com/Eder-PG/Microverse-5-Design-Teardown-Eder-Salvador/ft-heatmap/index.html

Any feedback is welcome

### CODE REVIEW

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team! :wave: 

### Great job thus far! :raised_hands: I can see you invested a lot of time in this. :100: The rightmost part of your design is blank on my screen and this is not in accordance with the following:

* [ ] It contains 2 articles, each takes half of the width. ([link to page](https://www.smashingmagazine.com/)) They do not take half of the width in your design.
* [ ] It contains 3 articles, each takes one third of the width (content is not important) ([link to page](https://www.smashingmagazine.com/)). The space on the right appears to be 1/5th which affects the size of other elements.
* [ ] It contains 1 article, which takes full width (content is not important) ([link to page](https://www.smashingmagazine.com/)). The article does not take the full width. 

Please see the images below for more clarification on the 3 points above.

**One**
screen

**Two**
screen1

**Three**
screen2

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

**Comment by Salvador-ON** [comment]
we fix the three issues fixing all the widths

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team, :wave:

### The width looks the same on my screen even though I made sure to purge the cache. Perhaps, you need to center the content?

screen

**Kindly submit for another code review once this has been fixed.** _Happy coding!_ :tada:

**Comment by Salvador-ON** [comment]
We fix the width of the main document that was causing the issue of the correct display set up the correct margin in each container

File: index.html [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+                <div class="smash-button"> <h4>Explore features</h4></div>              
+            </article>
+
+        </div>
+
+        <div class="more-articles-1"></div>
+
+    </main>
+
+    <aside>
```
Comment (addod19) [inline index.html:408]: I will recommend you use ```<footer>``` instead of ```<aside>``` to represent the exact content.

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :+1: 

Hi Team, great job so far. In order to get this project approved we need to fix the following comments:

- Kindly fix the errors returned by the CSS validator. [Link here](https://jigsaw.w3.org/css-validator)
- Kindly fix the horizontal scroll at the footer
- Kindly try to make your project look like the [original](https://www.smashingmagazine.com/)

Please fix the changes and submit another code review :slightly_smiling_face: 
Happy coding :computer: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Comment by Eder-PG** [comment]
Fixed scroll bar showing
Fixed width issues
Improved appearance of the page to look more similar to the base page

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :+1: 

Great job implementing the requested changes. Kindly fix the following to get your project approved:
- Each nav item has an icon, title and description
text
- Nav
nav
- Kindly pay attention to the background gradient
back

Make these changes and submit code review :+1: 
Good luck with your next submission:computer:

**Comment by Salvador-ON** [comment]
we add icon, title and description to the nav elements

about the second the gradient  it is not a requirement of this project to style  them to look like the original homepage

**Comment by Eder-PG** [comment]
added gradients and applied some more changes on the layout to mimick the original

**Review by addod19 — APPROVED** [review]
## Status: Approved :100: 

**Hi TEAM**, your project is good to go:airplane:  **Congratulations** on implementing the requested changes. :+1: 
You can now merge to master and submit the project completion form.
Good luck with your next project :slightly_smiling_face: 
Happy coding:computer:

**Pull request merged**